source/Grid/Grid.js

```diff
@@ -10,6 +10,7 @@ import {
 } from '../utils'
 import cn from 'classnames'
 import raf from 'raf'
+import getScrollbarSize from 'dom-helpers/util/scrollbarSize'
```

Comment:
I added a dependency bc it seemed better then copying the code, especially since `dom-helpers` is structured in a way that won't bring in extra code. https://github.com/react-bootstrap/dom-helpers/blob/master/src/util/scrollbarSize.js